### ISSUE TITLE
Prepare repository for Zenodo

### ISSUE THREAD

**Issue by peterdesmet** [enram/timamp-etl#32]
* [x] 1. Have final flyway settings: https://github.com/enram/vp-processing/pull/30
* [x] 2. Have final radars: https://github.com/enram/vp-processing/pull/31
* [x] 3. Merge flyway branch into master
* [x] 4. Remove all flyway related stuff from master
* [x] 5. Verify bookdown still uses example data
* [x] 6. Run bookdown one last time
* [x] 7. Combine all flyway CSV files into one: https://github.com/enram/vp-processing/tree/flyway/data/processed/flyway
* [x] 8. Generate flowviz data for flyway: https://github.com/enram/vp-processing/tree/flyway/data/flowviz/flyway
* [x] 9. Update README
* [ ] 10. Remove flyway branch

**Comment by peterdesmet** [comment]
Point 7: all processed flyway data have been combined into one file. It is too big for this repository, but it has been uploaded to Zenodo (under embargo for now). Extra changes:

* filename updated to: `vp_processed_70_radars_20160919_20161009.csv` ("processed" rather than "data")
* `NA` in the csv are blank values (cf. how the vp-processing will export the csv file)
* Row numbers have been removed (cf. how the vp-processing will export the csv file)
* The whole file has been sorted on radar_id, datetime, height, as if the processing for the whole date range.

I made sure the underlying data remained unchanged.
